README.md

```diff
@@ -7,3 +7,16 @@ This is a repo for studying purposes only.
 Contains a base project using OpenGL, GLEW and GLFW to build projects upon.
 
 SOIL and GLM, as well as a class for Shader compiling, are already included.
+
+## Instructions
+
+1. Open Visual Studio
+2. Click on "Open Project/Solution"
+3. Go to the folder where you downloaded this project
+4. Select "Sabertooth.sln"
+
+### If you have a Windows SDK error
+
+1. At the right hand side under "Solution Explorer" 
```

Comment:
Not necessarily at the right hand side. It might be moved by the user. Therefore, you should refer only to the "Solution Explorer" window (and how to open it, if needed).